coop_point_of_sale/models/pos_session.py

```diff
@@ -29,6 +29,26 @@ class PosSession(models.Model):
         string='Day (Search)', compute='_compute_date_search',
         multi='_date_search', store=True, index=True)
 
+    order_qty = fields.Integer(
+        compute='_compute_orders', string='Orders Qty', store=True)
+
+    amount_subtotal = fields.Monetary(
+        compute='_compute_orders', string='Amount Subtotal', store=True)
+
+    total_amount = fields.Monetary(
+        compute='_compute_orders', string='Transactions Total', store=True)
+
```

Comment:
```suggestion
    order_count = fields.Integer(
        compute='_compute_orders',
        string='Orders Count',
        store=True,
    )
    amount_untaxed = fields.Monetary(
        compute='_compute_orders',
        string='Amount Untaxed',
        store=True,
    )
    amount_total = fields.Monetary(
        compute='_compute_orders',
        string='Amount Total',
        store=True,
    )
```

To be consistent with the name in Odoo fields, please rename them as follows